datafusion/src/physical_plan/csv.rs

```diff
@@ -441,7 +441,7 @@ impl ExecutionPlan for CsvExec {
                 });
 
                 Ok(Box::pin(CsvStream::new(
-                    self.schema.clone(),
+                    self.projected_schema.clone(),
```

Comment:
The cause for failed tests `csv_query_window_with_empty_over` and `csv_query_window_with_order_by`. due to wrong schema provided.